products/cloudbuild/api.yaml

```diff
@@ -239,6 +239,13 @@ objects:
               The images are pushed using the builder service account's credentials.
               The digests of the pushed images will be stored in the Build resource's results field.
               If any of the images fail to be pushed, the build status is marked FAILURE.
+          - !ruby/object:Api::Type::String
+            name: 'timeout'
+            description: |
+              Amount of time that this build should be allowed to run, to second granularity. 
+              If this amount of time elapses, work on the build will cease and the build status will be TIMEOUT.
+              Default time is ten minutes.
```

Comment:
maybe write (600s) since the expected format is a time? I'd also put a line like "Expected format is number of seconds followed by s" or something else if it accepts different formats

Reply:
I'd also put a comment like "this must be greater than the sum of the timeouts for steps"